src/target/riscv/riscv-013.c

```diff
@@ -5367,6 +5368,14 @@ static enum riscv_halt_reason riscv013_halt_reason(struct target *target)
 		return RISCV_HALT_INTERRUPT;
 	case CSR_DCSR_CAUSE_GROUP:
 		return RISCV_HALT_GROUP;
+	case CSR_DCSR_CAUSE_OTHER:
+		if (get_field(dcsr, CSR_DCSR_EXTCAUSE) == 0) {
+			LOG_TARGET_INFO(target, "halted because of hart in a critical error state.");
```

Comment:
Since this is an error condition of the target, I would use at least warning level. Perhaps even an error?

```suggestion
			LOG_TARGET_WARNING(target, "halted because of hart in a critical error state.");
```

Reply:
Addressed